app/services/instrumentation.rb

```diff
@@ -0,0 +1,32 @@
+module Instrumentation
+  class << self
+    def append_to_log(args = {})
+      RequestStore.store[:custom_log_items] ||= {}
+      RequestStore.store[:custom_log_items].merge!(args) unless args.blank?
+      RequestStore.store[:custom_log_items]
+    end
+
+    def custom_log_items
+      append_to_log
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    def log(message, category = nil)
+      fail 'Block required' unless block_given?
+
+      started_at = Time.now.utc
```

Comment:
What do you about extracting lines 17 to 20 into a private method that does the timing? that way the method would be more concise and hopefully get rid of the rubocop warning.